website/docs/r/virtual_machine.html.markdown

```diff
@@ -172,6 +172,12 @@ A `boot_diagnostics` block supports the following:
 
 ---
 
+A `additional_capabilities` block supports the following:
```

Comment:
@venkey1000 could we also document that this exists [above](https://github.com/terraform-providers/terraform-provider-azurerm/pull/3860/files#diff-dc0e72edddc0d9ca0c01b859b58a226dR115)?

Reply:
I have commit the changes

Reply:
@tombuildsstuff after adding additional_capabilities in the last commit my build is failing. Can you please help me out.
Following is the change i have added in the last commit.
* `additional_capabilities` - (Optional) A `additional_capabilities` block.

Reply:
@tombuildsstuff can u please review the change of test case i have written for additional_capabilities?

Reply:
@tombuildsstuff Hi tom, can you add this feature to 1.32, as we have urgent requirement for this feature.